src/js/common/schemaform/widgets/TextWidget.jsx

```diff
@@ -13,7 +13,7 @@ export default function TextWidget(props) {
         maxLength={props.schema.maxLength}
         autoComplete={props.options.autocomplete || false}
         className={props.options.widgetClassNames}
-        value={props.value}
+        value={props.value || ''}
```

Comment:
Blank strings are converted to `undefined` in the onChange function, so this shouldn't cause any problems (and was how it worked before the recent change). Otherwise, it should have no effect.